app/controllers/claims/support/claim_windows_controller.rb

```diff
@@ -0,0 +1,72 @@
+class Claims::Support::ClaimWindowsController < Claims::Support::ApplicationController
+  before_action :set_claim_window, only: %i[show edit edit_check update remove destroy]
+  before_action :authorize_claim_window
+
+  def index
+    @claim_windows = Claims::ClaimWindow.order(starts_on: :desc)
+  end
+
+  def new
+    @claim_window = if params[:claims_claim_window]
+                      Claims::ClaimWindow.new(claim_window_params)
+                    else
+                      Claims::ClaimWindow.new
+                    end
+  end
+
+  def new_check
+    @claim_window = Claims::ClaimWindow::Build.call(claim_window_params:)
+
+    render :new unless @claim_window.valid?
+  end
+
+  def create
+    @claim_window = Claims::ClaimWindow::Build.call(claim_window_params:)
+    @claim_window.save!
+
+    redirect_to claims_support_claim_windows_path, flash: { success: "Claim window created" }
```

Comment:
This is the first use of the word "created" in our app. Should this not be "Claim window added"?
I would also suggest this go into `config/locales` rather than hard coded text.

Reply:
Good shout. This is a quick change. I'll address it now.